pkgs/by-name/ne/netclient/package.nix

```diff
@@ -1,34 +1,61 @@
-{ buildGoModule
+{ lib
+, buildGoModule
 , fetchFromGitHub
-, lib
-, libX11
+, installShellFiles
+, makeWrapper
+, nix-update-script
 , stdenv
-, darwin
+, systemd
 }:
-
 buildGoModule rec {
   pname = "netclient";
-  version = "0.22.0";
+  version = "unstable-2024-01-26";
 
   src = fetchFromGitHub {
-    owner = "gravitl";
+    # TODO: switch to upstream after https://github.com/gravitl/netclient/pull/682
+    #owner = "gravitl";
+    owner = "nazarewk";
```

Comment:
Might be better to have this PR as patches and keep the upstream repo? Not sure what's generally preferred but that's what I usually see.

Reply:
can do that at this point (got everything as much to work as possible), but it's extremely tedious to update whenever code change is required.